JSDemos/Demos/PivotGrid/OLAPDataSource/description.md

```diff
@@ -1 +1,20 @@
-The PivotGrid supports OLAP services (MS&nbsp;SQL Server Analysis Services). This allows you to&nbsp;deliver solutions that can easily analyze millions of&nbsp;data rows within your app. In&nbsp;this demo, the PivotGrid uses **XmlaStore** to&nbsp;connect the component to&nbsp;a&nbsp;sample remote OLAP service. The Field Chooser is&nbsp;configured to&nbsp;show the search box to&nbsp;speed up&nbsp;choosing the required field.
\ No newline at end of file
+The PivotGrid component supports OLAP services (Microsoft SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
+
+## Configure the Store
+To connect your component to an OLAP storage, use [XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/). Specify the following properties to configure it:    
+
+- [url](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#url)        
+The OLAP server's URL.
+
+- [catalog](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#catalog)        
+The initial catalog that contains the OLAP cube.
+
+- [cube](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#cube)       
+The name of the OLAP cube to use from the catalog.
+
+Pass the XmlaStore to the [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) property of the [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) component. Assign the component to the PivotGrid's [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property.
```

Comment:
The verb "pass" looks strange to me when used with a property. You **pass** an object or value into a certain method as a parameter, for example. However, you do **assign** an object to a property. I believe you did this to avoid repetition in the second sentence, however, this is not a reason to use an inappropriate word. There are better ways to avoid repetition.

From what I gather, the described operation consists of two steps:
1. Assign a previously configured XmlaStore object to the `PivotGridDataSource.store` property.
2. Assign the `PivotGridDataSource` component to the PivotGrid component's `dataSource` property.

Reply:
Yes, the two steps you described are correct. How about the following: Assign the XmlaStore to the [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) property of the [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) component. Then assign the **PivotGridDataSource** component to the PivotGrid's [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property.

Reply:
Yeah, that would do indeed.